src/Common/src/CoreLib/System/Threading/Tasks/IValueTaskSource.cs

```diff
@@ -0,0 +1,83 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Would there be mirroring issues if you include it here while it is already in coreclr? cc @Anipik 

Also the path here is Threading/Tasks/IValueTaskSource.cs but in coreclr it was Threading/Tasks/Sources/IValueTaskSource.cs

Reply:
This is a stray file and should be deleted.  The file has already mirrored to corefx, it's just at a different path; this is an old copy at a different location that somehow snuck in.

Reply:
Deleted.